src/ocean/Ocean.H

```diff
@@ -63,6 +63,7 @@ public:
     using LandMask  = Utils::MaskStruct;
 
 protected:
+    Teuchos::ParameterList params_;
```

Comment:
This should probably be an RCP so the Belos parameterlist also sets the used and default parameters. Even better if this can be removed.

Reply:
I don't think it can be removed if we wanna support querying the current parameter values. On the other hand, we could "not support" that, I guess?

Reply:
If you implement my previous suggestion (using params_ to initialize result), then we have to keep it anyway.